kratos/utilities/reduction_utilities.h

```diff
@@ -28,8 +28,35 @@
 
 namespace Kratos
 {
+
+/** @brief Helper class for null-initializiation
+ */
+template <class TObjectType>
+struct NullInitialized
+{
+    static TObjectType Get()
+    {
+        return TObjectType();
+    }
+};
+
+template <class TValueType, std::size_t ArraySize>
+struct NullInitialized<array_1d<TValueType,ArraySize>>
+{
+    static array_1d<TValueType,ArraySize> Get()
+    {
+        array_1d<TValueType,ArraySize> array;
+        std::fill_n(array.begin(), ArraySize, 0);
+        return array;
+    }
+};
```

Comment:
I agree that for now this shouldn't be public

can you put it into an anonymous namespace?
I think that would be the easiest solution

Reply:
Ah I see, I thought @RiccardoRossi was referring to the member variable being public. Sure, moving it to a separate namespace makes sense.

Reply:
@philbucher did you mean something like ```namespace Internals```?
I looked up what an anonymous namespace is and don't see why it would be useful in a header.

Reply:
yes sorry you are right